<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>916</width>
    <height>700</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>1000</width>
      <height>700</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideLeft</enum>
    </property>
    <property name="tabsClosable">
     <bool>true</bool>
    </property>
    <property name="movable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>-10</y>
      <width>121</width>
      <height>661</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="bkg_icons.qrc">:/invagate/icons/left_slide.png</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>916</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="quitAct"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>工作</string>
    </property>
    <addaction name="inStoreAct"/>
    <addaction name="outStoreAct"/>
    <addaction name="chkGoodAct"/>
    <addaction name="recExcepAct"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>日志</string>
    </property>
    <addaction name="wDiaryAct"/>
    <addaction name="diaryHisAct"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>统计</string>
    </property>
    <addaction name="mthrAct"/>
    <addaction name="nxtMthrAct"/>
    <addaction name="brknRateAct"/>
    <addaction name="rnkComAct"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>账户管理</string>
    </property>
    <addaction name="loginAct"/>
    <addaction name="logoutAct"/>
    <addaction name="newAcntAct"/>
    <addaction name="chpasswdAct"/>
    <addaction name="bkpasswdAct"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="manDocAct"/>
    <addaction name="faqAct"/>
   </widget>
   <widget class="QMenu" name="menu_6">
    <property name="title">
     <string>查看</string>
    </property>
    <addaction name="curStorAct"/>
    <addaction name="lookGoodByPosi"/>
    <addaction name="lookGoodByInfo"/>
   </widget>
   <widget class="QMenu" name="menu_7">
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>超时</string>
    </property>
    <addaction name="overtime"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu"/>
   <addaction name="menu_6"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
   <addaction name="menu_7"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="wDiaryAct"/>
   <addaction name="diaryHisAct"/>
   <addaction name="separator"/>
   <addaction name="mthrAct"/>
   <addaction name="nxtMthrAct"/>
   <addaction name="brknRateAct"/>
   <addaction name="rnkComAct"/>
   <addaction name="separator"/>
   <addaction name="loginAct"/>
   <addaction name="logoutAct"/>
   <addaction name="overtime"/>
   <addaction name="separator"/>
  </widget>
  <action name="quitAct">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="inStoreAct">
   <property name="text">
    <string>入库</string>
   </property>
  </action>
  <action name="outStoreAct">
   <property name="text">
    <string>出库</string>
   </property>
  </action>
  <action name="chkGoodAct">
   <property name="text">
    <string>验货</string>
   </property>
  </action>
  <action name="recExcepAct">
   <property name="text">
    <string>异常事件</string>
   </property>
  </action>
  <action name="wDiaryAct">
   <property name="text">
    <string>写日志</string>
   </property>
  </action>
  <action name="diaryHisAct">
   <property name="text">
    <string>历史日志</string>
   </property>
  </action>
  <action name="mthrAct">
   <property name="text">
    <string>本月吞吐</string>
   </property>
  </action>
  <action name="nxtMthrAct">
   <property name="text">
    <string>下月吞吐</string>
   </property>
  </action>
  <action name="brknRateAct">
   <property name="text">
    <string>储货破损率</string>
   </property>
  </action>
  <action name="rnkComAct">
   <property name="text">
    <string>排序公司</string>
   </property>
  </action>
  <action name="loginAct">
   <property name="text">
    <string>登陆</string>
   </property>
  </action>
  <action name="logoutAct">
   <property name="text">
    <string>退出登陆</string>
   </property>
  </action>
  <action name="newAcntAct">
   <property name="text">
    <string>新建账号</string>
   </property>
  </action>
  <action name="chpasswdAct">
   <property name="text">
    <string>修改密码</string>
   </property>
  </action>
  <action name="bkpasswdAct">
   <property name="text">
    <string>密码找回</string>
   </property>
  </action>
  <action name="manDocAct">
   <property name="text">
    <string>使用说明</string>
   </property>
  </action>
  <action name="faqAct">
   <property name="text">
    <string>疑问解答</string>
   </property>
  </action>
  <action name="curStorAct">
   <property name="text">
    <string>当前仓储量</string>
   </property>
  </action>
  <action name="lookGoodByPosi">
   <property name="text">
    <string>按位置查看货物</string>
   </property>
   <property name="toolTip">
    <string>按位置查看货物</string>
   </property>
  </action>
  <action name="lookGoodByInfo">
   <property name="text">
    <string>按信息查看货物</string>
   </property>
  </action>
  <action name="overtime">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>超时提醒</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="bkg_icons.qrc"/>
 </resources>
 <connections/>
</ui>
